addons/trenches/CfgVehicles.hpp

```diff
@@ -34,6 +59,12 @@ class CfgVehicles {
         descriptionShort = CSTRING(EnevlopeSmallDescription);
         model = QUOTE(PATHTOEF(apl,ace_envelope_small4.p3d));
         scope = 2;
+        GVAR(diggingDuration) = 20;
```

Comment:
Defining the data here when the rest of the placement data is hard-coded in `preInit` is a bit inconsistent. Of course using the config is better, so we should probably migrate the rest of the data here eventually.

Ideally we could organize the data so more trench types could be added in config by third parties.